central/buildbot.py

```diff
@@ -22,18 +22,18 @@ def make_netstring(s):
     return str(len(s)).encode('ascii') + b':' + s + b','
 
 
-def make_build_request(repo, pr_id, job_id, baserev, headrev, patch, who,
+def make_build_request(repo, pr_id, job_id, baserev, headrev, who,
                        comment):
     """Creates a build request binary blob in the format expected by the
     buildbot."""
 
     request_dict = {
-        'branch': '',
+        'branch': 'refs/pull/%d/merge' % pr_id,
```

Comment:
Yeah, I'm not sure. Is there a way to test it?

It might be better handled in central, which currently ignores a null mergeable value: https://developer.github.com/v3/pulls/#get-a-single-pull-request